go.mod

```
@@ -39,8 +39,8 @@ require (
 	gomodules.xyz/jsonpatch/v2 v2.2.0
 	gonum.org/v1/gonum v0.11.0
 	gopkg.in/yaml.v2 v2.4.0
-	istio.io/api v0.0.0-20220304035241-8c47cbbea144
-	istio.io/client-go v1.12.5
+	istio.io/api v0.0.0-20220512181135-e8ec1e1d89de
+	istio.io/client-go v1.14.0
```

Should we go for `v1.14.1` now?
https://github.com/istio/client-go/releases/tag/1.14.1
https://github.com/istio/api/releases/tag/1.14.1

```suggestion
	istio.io/api v1.14.1
	istio.io/client-go v1.14.1
```

---

I can try again, but as the git tag is still not there I assume that it will fail again as the version will not be found. Will report back of the results

---

```
cloud.google.com/go/compute/metadata: istio.io/api@v1.14.1: reading istio.io/api/go.mod at revision v1.14.1: unknown revision v1.14.1
cloud.google.com/go/compute/metadata: istio.io/client-go@v1.14.1: reading istio.io/client-go/go.mod at revision v1.14.1: unknown revision v1.14.1
```
For `api`, we can utilise the proper version corresponding to what `client-go` uses as before, but `client-go` should simply be available...